File: test_tiger.py
import requests
import json
import sqlite3
import pymysql

conn = pymysql.connect(host='localhost', user='root', password='', db='smsbyog')
cursor = conn.cursor()
cursor.execute("SELECT api_url, api_key FROM api_detail WHERE id='8'")
row = cursor.fetchone()
api_url = row[0]
api_key = row[1]

url = f"{api_url}/stubs/handler_api.php?api_key={api_key}&action=getPrices"
response = requests.get(url, verify=False)
data = response.json()
print("Total countries in prices:", len(data))
first_country = list(data.keys())[0]
print(f"Country {first_country} has {len(data[first_country])} services")

File: test_tiger.php
<?php
include 'include/config.php';
$sql4 = mysqli_query($conn, "SELECT * FROM api_detail WHERE id='8'");
$api_data = mysqli_fetch_assoc($sql4);
$api_url = $api_data['api_url'];
$api_key = $api_data['api_key'];

$url = "{$api_url}/stubs/handler_api.php?api_key={$api_key}&action=getPrices";
$ch = curl_init($url);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
$raw = curl_exec($ch);
curl_close($ch);
$data = json_decode($raw, true);

echo "Total countries in prices: " . count($data) . "\n";
// Print structure of first country
foreach ($data as $country => $services) {
    echo "Country: $country\n";
    echo "Total services: " . count($services) . "\n";
    break;
}
